roles/virtualbmc/tasks/main.yml

```diff
@@ -67,14 +67,14 @@
           server_port={{ cifmw_virtualbmc_daemon_port }}
         mode: "0644"
 
-    - name: Create and start vbmc container
-      vars:
-        dest_dir: "{{ cifmw_virtualbmc_sshkey_path | dirname }}"
-      containers.podman.podman_container:
-        image: "{{ cifmw_virtualbmc_image }}"
-        name: "{{ cifmw_virtualbmc_container_name }}"
-        network: host
-        state: started
-        volumes:
-          - "{{ cifmw_virtualbmc_sshkey_path }}:{{ cifmw_virtualbmc_ipmi_key_path }}:ro,Z"
-          - "{{ dest_dir }}/virtualbmc.conf:/etc/virtualbmc/virtualbmc.conf:ro,Z"
+- name: Create and start vbmc container
+  vars:
+    dest_dir: "{{ cifmw_virtualbmc_sshkey_path | dirname }}"
+  containers.podman.podman_container:
+    image: "{{ cifmw_virtualbmc_image }}"
+    name: "{{ cifmw_virtualbmc_container_name }}"
+    network: host
+    state: started
+    volumes:
+      - "{{ cifmw_virtualbmc_sshkey_path }}:{{ cifmw_virtualbmc_ipmi_key_path }}:ro,Z"
+      - "{{ dest_dir }}/virtualbmc.conf:/etc/virtualbmc/virtualbmc.conf:ro,Z"
```

Comment:
tested, and it worked!

Reply:
I moved that change in a dedicated patch, alongside another tweak to that container:
https://github.com/openstack-k8s-operators/ci-framework/pull/1405